Gemfile

```diff
@@ -147,9 +148,7 @@ gem 'mail', '~> 2.7.1'
 gem 'feedjira', '~> 3.2'
 gem 'bootsnap', require: false
 gem 'rails_semantic_logger', '~> 4.14'
-gem 'dotenv', '~> 3.1'
 gem 'whenever', '~> 1.0', require: false
-gem 'puma', '~> 5.6'
```

Comment:
We need puma 5.6 on all environments. Can you revert your changes to the Gemfile? They're not related to the redis issue.

Reply:
@stevenbarragan  - There seems to be duplicates between all branches. I found puma within development, test blocks and outside of those blocks in the Gemfile. I will put this one back but this Gemfile will still have a change to remove the dups.